// ajv-keywords/keywords/index.js
'use strict';

module.exports = {
  'instanceof': require('ajv-keywords/keywords/instanceof'),
  range: require('ajv-keywords/keywords/range'),
  regexp: require('ajv-keywords/keywords/regexp'),
  'typeof': require('ajv-keywords/keywords/typeof'),
  dynamicDefaults: require('ajv-keywords/keywords/dynamicDefaults'),
  allRequired: require('ajv-keywords/keywords/allRequired'),
  anyRequired: require('ajv-keywords/keywords/anyRequired'),
  oneRequired: require('ajv-keywords/keywords/oneRequired'),
  prohibited: require('ajv-keywords/keywords/prohibited'),
  uniqueItemProperties: require('ajv-keywords/keywords/uniqueItemProperties'),
  deepProperties: require('ajv-keywords/keywords/deepProperties'),
  deepRequired: require('ajv-keywords/keywords/deepRequired'),
  formatMinimum: require('ajv-keywords/keywords/formatMinimum'),
  formatMaximum: require('ajv-keywords/keywords/formatMaximum'),
  patternRequired: require('ajv-keywords/keywords/patternRequired'),
  'switch': require('ajv-keywords/keywords/switch'),
  select: require('ajv-keywords/keywords/select'),
  transform: require('ajv-keywords/keywords/transform')
};
